portfolio/src/main/webapp/index.html

```diff
@@ -27,13 +27,13 @@ <h1>Tal's Portfolio</h1>
       
      
       
-      <form id = 'notLoggedIn' method="GET" >
+      <form id = 'notLoggedIn' method="POST" >
         <button id = "button">Login to Comment</button>
       </form>
      
       
       
-      <form id='logOut' method="GET" >
+      <form id='logOut' method="POST" >
```

Comment:
Changed from GET to POST because it looks like comments forms did that. Not exactly sure what the best way to do it is